decidim-core/app/controllers/decidim/pages_controller.rb

```diff
@@ -23,21 +23,18 @@ def page_finder
       @page_finder ||= Decidim::PageFinder.new(params[:id], current_organization)
     end
 
-    def users
-      @users ||= Decidim::User.where(organization: current_organization)
-    end
-
-    # This should be deleted once the statistics are done properly.
-    def participatory_processes
```

Comment:
Wasn't this used in the homepage to list some active participatory processes? Right before the stats

https://www.decidim.barcelona/